topics.json

```
@@ -0,0 +1,6 @@
+{
+    "azurerm-": "azure",
+    "aws-": "aws",
+    "gcp-": "google-cloud",
+    "tf-": "terraform"
```

We probably ought to capture the various flavors of IaC repo from here: https://github.com/launchbynttdata/common-platform-documentation/blob/main/standards/common-development/git/repository/naming-schemes/iac-repository-names.md

```suggestion
    "tf-": "terraform",
    "helm-": "helm",
    "module_primitive-": "primitive",
    "module_collection": "collection",
    "module_reference-": "reference_architecture",
    "module_library": "library",
    "chart_library": "helm_library",
    "chart_deployment": "helm_deployment",
    "chart_umbrella": "helm_umbrella"
```

I'm flexible on the final topic names but figured this is a reasonable starting point.